contracts/seed/Seed.sol

```diff
@@ -53,25 +53,30 @@ contract Seed {
     // Contract logic
     bool      public closed;
     bool      public paused;
-    uint256   public totalLockCount;
+    uint256   public totalLockCount;   // Total locks that have been created. Each user can have only one lock.
+    uint256   public seedRemainder;    // Amount of seed tokens remaining to be distributed
```

Comment:
therefore by definition, does not include fees

Reply:
Have added this to a new Issue. #160. This change needs some test changes too, so it's better to move it to a new branch because this PR is very big now.